### PR TITLE
PLAT-245: remove old func tests

### PR DESCRIPTION
**Pull request by sexyhamster**
Removed all the redundant func tests that were testing Pl…atform 1.0 behaviour

### CODE REVIEW

File: ledger-core/v2/application/functest/get_info_test.go [hunk 1]
```diff
@@ -1,22 +0,0 @@
...
-
-package functest
-
-import (
-	"testing"
-
-	"github.com/stretchr/testify/require"
-)
-
-func TestGetInfo(t *testing.T) {
```
Comment (ElSacramento) [inline ledger-core/v2/application/functest/get_info_test.go:16]: It works now (jfyi).

Comment (ruz) [reply ledger-core/v2/application/functest/get_info_test.go:16]: @ElSacramento, do we need this test? It's not clear from your comment.

Comment (ElSacramento) [reply ledger-core/v2/application/functest/get_info_test.go:16]: We have discussed on one of the team meetings, that it's ok to delete not working tests, but this test works. I'm not sure that it's useful now for platform 2.0, but while it works maybe we can leave it? Just a thought.

Comment (sexyhamster) [reply ledger-core/v2/application/functest/get_info_test.go:16]: We discussed in Plat 246, that we don't need this contract, so it will be removed in #69

Comment (ruz) [reply ledger-core/v2/application/functest/get_info_test.go:16]: took closer look. agree.

**Pull request merged**